.github/workflows/main.yaml

```diff
@@ -47,5 +47,7 @@ jobs:
       - name: Run Tests
         run: pdm test
         env:
-          EMAIL: ${{ secrets.GRADESCOPE_CI_STUDENT_EMAIL }}
-          PASSWORD: ${{ secrets.GRADESCOPE_CI_STUDENT_PASSWORD }}
+          GRADESCOPE_CI_STUDENT_EMAIL: ${{ secrets.GRADESCOPE_CI_STUDENT_EMAIL }}
```

Comment:
I forget, where are we storing these secrets variables?

Reply:
They are stored in GitHub secrets in the repo settings. I'll also share the credentials on slack for local testing

Reply:
Thanks Calvin! I'll wait for your dm

Reply:
Ctaching up with the missed Reviews.